package-lock.json

```diff
@@ -2992,7 +2992,8 @@
             },
             "ansi-regex": {
               "version": "2.1.1",
-              "bundled": true
+              "bundled": true,
+              "optional": true
```

Comment:
Do you have any idea for the `optional` appear?
I read some stuff here
https://stackoverflow.com/questions/54411377/package-lock-json-file-package-with-optional-true

It might have happened due to different nodejs version that we are using, I'm not sure. 
In this case, maybe we need to use something like this https://nitayneeman.com/posts/standardizing-node.js-version-in-an-npm-package/

Another assumption is that you removed and added something as I understood based on this stackoverflow link.
Someone said to run:

```
rm -rf package-lock.json node_modules/
npm install
```

Do you have any ideas regarding it?

Reply:
Nop...
Just installed the App after I've cloned it...(used npm install)

Reply:
My Node version is v12.14.0

Reply:
My npm version is 6.13.4